src/api/test/functional/webui/project_controller_test.rb

```diff
@@ -245,6 +245,56 @@ def test_add_and_modify_repo
                    :tag => "arch", :content => "x86_64"
   end
 
+  def test_save_meta
+    use_js
+
+    login_adrian
+    visit(project_show_path(project: "home:adrian"))
+
+    # Test reading meta data
+    click_link("Advanced")
+    click_link("Meta")
+    # Note that textarea#editor_0 is a hidden element.
+    # This isn't ideal for a test, but best we can do to test this part of the ui
+    assert find(:css, "textarea#editor_0", visible: false).
+      text(:all).include?("<title>adrian's Home Project</title>")
+
+    # Test writing valid meta data
+    xml = <<-XML.gsub(/(?:\s*\n|^\s*)/, '') # evaluate_script fails otherwise
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.